.eslintrc

```diff
@@ -15,7 +15,11 @@
     "plugins": ["@typescript-eslint", "import"],
     "extends": ["standard", "plugin:import/errors", "plugin:import/warnings"],
     "rules": {
+        "no-use-before-define": "off",
```

Comment:
When using eslint and typescript eslint, you have to turn off some regular eslint rules.